src/Services/QueuedJobService.php

```diff
@@ -66,6 +66,37 @@ class QueuedJobService
      */
     private static $stall_threshold = 3;
 
+    /**
+     * How early stuck jobs will become eligible for automated retry processing (minutes)
+     *
+     * @config
+     * @var int
+     */
+    private static int $job_retry_sentinel = 1;
+
+    /**
+     * How many jobs are eligible to be automatically retried per one health check
```

Comment:
```suggestion
     * How many jobs are eligible to be automatically retried per health check
```
"one" is implied by "per"